articles/tech-resources/ios-universal-links-self-serve.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using iOS Universal Links with SparkPost"
+description: "A guide to including iOS universal links in your SparkPost-delivered email"
+---
+
+## Introduction
+
+Universal links are a mechanism in iOS 9 and later that allows you to link deeply into your iOS apps from web pages and HTML email. To use universal links, you must publish the URLs you'd like your iOS app to handle and then add support for them into the app itself. When your users receive an email containing a universal link, iOS then recognises the domain and path of the URL and triggers your app. As a result, it's important to keep the URLs in your HTML emails intact to ensure your universal links function correctly.
+
+This article describes the various ways you can use universal links with email sent through your SparkPost account.
+
+[Apple's Universal Links](https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html) documentation explains the iOS app mechanisms in detail. As such, we will not restate those details here.
+
+## Prerequisites
+
+- an iOS app with universal link support
+- an apple-app-site-association file hosted on your own domain
+
+#### Sidebar: SparkPost Click Tracking
+
+In the discussion below, it might be useful to understand how the SparkPost click tracking feature works. In essence, when the feature is enabled and SparkPost encounters an anchor tag in an HTML email, it will replace the `href` attribute with a new URL pointing to it's click tracking service. When your recipient clicks that link, the click tracker receives the request, records the "click" and redirects the recipient to your original URL.
+
+You can customize the domain SparkPost uses in your tracked links by setting up a [custom tracking domain](https://app.sparkpost.com/account/tracking-domains). This is useful from a branding and reputation standpoint and it also offers a mechanism for including universal links in your HTML email.
```

Comment:
Do you think linking to the app is sufficient? Or should we link to the [support doc](https://www.sparkpost.com/docs/tech-resources/enabling-multiple-custom-tracking-domains/)?